packages/yoga/src/Checkbox/native/Checkbox.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import { bool, func, string } from 'prop-types';
+import styled from 'styled-components';
+import { TouchableWithoutFeedback } from 'react-native';
+import CheckMark from './CheckMark';
+
+const CheckboxWrapper = styled.View`
+  width: 100%;
+  flex-wrap: nowrap;
+  align-items: center;
+`;
+
+const Label = styled.Text(
+  ({
+    theme: {
+      components: { checkbox },
+    },
+  }) => `
+  padding-left: ${checkbox.label.padding.left}px;
+  font-size: ${checkbox.label.font.size}px;
+  color: ${checkbox.label.font.color};
+`,
+);
+
+const HelperWrapper = styled.View(
+  ({
+    theme: {
+      components: { checkbox },
+    },
+  }) => `
+  width: 100%;
+  flex-wrap: nowrap;
+  margin-top: ${checkbox.helper.margin.top}px;
+`,
+);
+
+const Helper = styled.Text(
+  ({
+    error,
+    theme: {
+      components: { checkbox },
+    },
+  }) => `
+  font-size: ${checkbox.helper.font.size}px;
+  color: ${
+    error
+      ? `${checkbox.helper.selected.font.color};`
+      : `${checkbox.helper.font.color}`
+  }
+`,
+);
+
+const Checkbox = ({
+  label,
+  helper,
+  disabled,
+  checked,
+  error,
+  onChange,
+  ...rest
+}) => {
+  const [pressed, setPressed] = useState(false);
+  return (
+    <>
+      <TouchableWithoutFeedback
+        accessibilityRole="checkbox"
+        onPress={() => !disabled && onChange()}
+        onPressIn={() => !disabled && setPressed(true)}
+        onPressOut={() => !disabled && setPressed(false)}
+        {...rest}
+      >
```

Comment:
Why not use the [withTouchable](https://github.com/Gympass/yoga/blob/master/packages/yoga/src/Button/native/withTouchable.jsx) HoC?

Reply:
We have tested and in this case is not applicable.